src/components/ui/QuestionBox/QuestionBox.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { StyledQuestionBox } from "./QuestionBox.style";
+import dummyData from "../../../api/dummyData";
+import { AnswerButton } from "../QuestionBox/QuestionBox.style";
+
+const QuestionBox = () => {
+  const [answerColors, setAnswerColors] = useState(
+    Array(4).fill("var(--color-for-pre-answer)"),
+  );
+  const [isAnswered, setIsAnswered] = useState(false);
+
+  const questionData = dummyData[0].questions[0];
+
+  const handleAnswerClick = (index) => {
+    if (isAnswered) return; // If already answered, do nothing
+
+    const updatedColors = [...answerColors];
+
+    if (index === questionData.correctAnswer) {
+      updatedColors[index] = "var(--color-for-correct-answer)";
```

Comment:
we should rename our css variable colors to be reusable. Green, blue, red etc. For example the red and green in your assignment is the same red and green for the toast message

Reply:
ok thx,
I change it :
![image](https://github.com/user-attachments/assets/a5bf088d-4d5f-4765-ad6e-ff880f9ec453)